suites/message/message_application.go

```diff
@@ -46,13 +46,41 @@ func MessageTest_MessageApplicationEdgecases(t *testing.T, factory state.Factori
 			td.MessageProducer.Transfer(alice, alice, chain.Value(transferAmnt), chain.Nonce(0), chain.GasPrice(10), chain.GasLimit(1)),
 			exitcode.SysErrOutOfGas)
 
-		// Expect Message application to fail due to lack of gas when sender address is unknown
+		// Expect Message application to fail due to lack of gas when sender is unknown
 		unknown := utils.NewIDAddr(t, 10000000)
 		td.ApplyFailure(
 			td.MessageProducer.Transfer(alice, unknown, chain.Value(transferAmnt), chain.Nonce(0), chain.GasPrice(10), chain.GasLimit(1)),
 			exitcode.SysErrOutOfGas)
 	})
 
+	t.Run("fail not enough gas to cover account actor creation", func(t *testing.T) {
+		td := builder.Build(t)
+		defer td.Complete()
+
+		alice, _ := td.NewAccountActor(drivers.SECP, aliceBal)
+		aliceNonce := uint64(0)
+		aliceNonceF := func() uint64 {
+			defer func() { aliceNonce++ }()
+			return aliceNonce
+		}
+		newAccountA := utils.NewSECP256K1Addr(t, "1")
+
+		// get the "true" gas cost of applying the message
```

Comment:
Note: the true gas cost cannot be determined by applying a message, this is an aproximation. Chain-validation uses a default gas limit of [`1,000,000`](https://github.com/filecoin-project/chain-validation/pull/193/files#diff-fdc8ab8a15ed13b6f5c5724adf12dadeL22) and a default gas price of [`1`](https://github.com/filecoin-project/chain-validation/pull/193/files#diff-fdc8ab8a15ed13b6f5c5724adf12dadeR23). These values are encoded into a message and since they are large they increase the size of the message on chain and thus its cost.